.gitignore

```
@@ -9,3 +9,4 @@ dist
 build
 lib
 test-results.json
+.history
```

Let's add a new line to the end of this file.

```suggestion
.history

```